ui/app/styles/components/replication-state-text.scss

```diff
@@ -0,0 +1,7 @@
+.title-link {
```

Comment:
You know me. Avoid component css files if possible. This looks like it could be a helper or core class instead. Hit me up if you need some help navigating that.

Reply:
I agree! I wasn't sure if it warranted a global helper since it felt a little specific to the replication card but this could be a pattern we use in the future